tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/core/TokenTagger.scala

```diff
@@ -64,7 +64,18 @@ case object LexicalPropertiesTagger extends TokenTagger {
     } map {
       _ => 'existsNum
     }
-    (Seq(firstLetterCapital, existsCapital, allCaps, existsNumber) map { maybeFeat =>
+    def parseDouble(s: String) = try { Some(s.toDouble) } catch { case _: Throwable => None }
+    val looksLikeDouble = parseDouble(tokStr) map {
+      _ => 'looksLikeDouble
+    }
+    val looksLikeGeoLabel =
```

Comment:
`Geo` is almost always abbreviation for "geographic". I really wondered for an instant what we're doing with geo stuff. Perhaps rename to `Geom`?